onadata/apps/main/tests/test_custom_context_processors.py

```diff
@@ -1,15 +1,9 @@
 # coding: utf-8
-from django.conf import settings
 from django.test import TestCase
 from onadata.apps.main.context_processors import site_name
 
 
 class CustomContextProcessorsTest(TestCase):
     def test_site_name(self):
         context = site_name(None)
-        self.assertEqual(context, {'SITE_NAME': 'example.com'})
-        restore_site_id = settings.SITE_ID
-        settings.SITE_ID = 100
-        context = site_name(None)
-        self.assertEqual(context, {'SITE_NAME': 'example.org'})
-        settings.SITE_ID = restore_site_id
+        self.assertEqual(context, {'SITE_NAME': 'kc.kobotoolbox.org'})
```

Comment:
It does not pass locally (because my domain name is not `kc.kobotoolbox.org`) 
I guess we could use something like:

```python
from django.test import override_settings

class CustomContextProcessorsTest(TestCase):
    @override_settings(KOBOCAT_PUBLIC_SUBDOMAIN='kc.domain.tld')
    def test_site_name(self):
         context = site_name(None)
         self.assertEqual(context, {'SITE_NAME': 'kc.domain.tld'})
```
```

Reply:
I tested by setting KOBOCAT_PUBLIC_SUBDOMAIN to another value. Test failed. With recent commit, it now passes. The specific override needs to be "KOBOCAT_PUBLIC_HOSTNAME"